tests/test_comms.py

```diff
@@ -34,10 +34,10 @@ def test_init_comms(self):
 
         for ctx, n_jobs, order_tasks in product(test_ctx, [1, 2, 4], [False, True]):
             comms = WorkerComms(ctx, n_jobs, order_tasks)
-            event_type = type(ctx.Event())
-            lock_type = type(ctx.Lock())
-
             with self.subTest('__init__ called', ctx=ctx, n_jobs=n_jobs, order_tasks=order_tasks):
+                condition_type = type(ctx.Condition(ctx.Lock()))
```

Comment:
Nope. Most multiprocessing classes are actually factory methods:
```
In [2]: c = Condition()
time: 1.93 ms (started: 2023-03-07 13:30:31 +01:00)

In [3]: isinstance(c, Condition)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Input In [3], in <cell line: 1>()
----> 1 isinstance(c, Condition)

TypeError: isinstance() arg 2 must be a type or tuple of types
time: 226 ms (started: 2023-03-07 13:30:35 +01:00)

In [4]: Condition??
Signature: Condition(lock=None)
Source:   
    def Condition(self, lock=None):                <-- See the 'def'
        '''Returns a condition object'''
        from .synchronize import Condition
        return Condition(lock, ctx=self.get_context())
File:      ~/miniconda3/envs/mpire/lib/python3.9/multiprocessing/context.py
Type:      method
```

Reply:
I could import the actual classes from `multiprocessing.synchronize` though. I think I will do that instead

Reply:
I'm going to leave this as is, as the types for threading, multiprocessing and `multiprocess` are all different, which makes the tests a bit ugly. This will do